src/main/kotlin/com/msg/gauth/domain/client/presentation/ClientController.kt

```diff
@@ -43,7 +44,10 @@ class ClientController(
     }
 
     @PatchMapping("/{id}")
-    fun updateClient(@PathVariable id: Long, @RequestBody clientUpdateDto: ClientUpdateReqDto): ResponseEntity<Void>{
+    fun updateClient(
+        @PathVariable id: Long,
+        @RequestBody clientUpdateDto: ClientUpdateReqDto
+    ): ResponseEntity<Void>{
```

Comment:
```suggestion
    ): ResponseEntity<Void> {
```
스타일 따라서 공백 넣어주면 좋을거같아요

Reply:
이걸 코파일럿ㅇ이